.github/workflows/build-and-test.yml

```diff
@@ -291,10 +287,7 @@ jobs:
             - name: Install dependencies
               shell: bash -l {0}
               run: |
-                  conda install nomkl # make sure numpy w/out mkl
-                  conda install pytest requests build
-                  conda install $(python -c "import toml;print(' '.join(toml.load('pyproject.toml')['build-system']['requires']))")
```

Comment:
`conda` doesn't understand `pip`'s environment markers that I added in the build-system requirements. But this line isn't needed any more because `build` will automatically install them.